Services/FileService/Broker/Requests/IGetFileRequest.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace LT.DigitalOffice.Broker.Requests
+{
+    /// <summary>
+    /// Represents request for GetFileConsumer in MassTransit logic.
+    /// </summary>
+    public interface IGetFileRequest
+    {
+        Guid FileId { get; }
+    }
```

Comment:
In UserService and ProjectService this field is named "Id". No conflicts happened?

Reply:
fixed